NuGet.config

```diff
@@ -7,13 +7,13 @@
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
-    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
   </packageSources>
   <packageSourceMapping>
     <packageSource key="dotnet-eng">
       <package pattern="MicroBuild.Core.Sentinel" />
       <package pattern="Microsoft.DotNet.Arcade.Sdk" />
       <package pattern="Microsoft.DotNet.Build.Tasks.Feed" />
```

Comment:
Please add "<package pattern="Microsoft.DotNet.Maestro.Tasks" />" under the dotnet-eng source mapping.